parm/wflow/verify_det.yaml

```diff
@@ -136,49 +136,60 @@ metatask_GridStat_REFC_RETOP_all_mems:
     task_run_MET_PointStat_vx_#FIELD_GROUP#_mem#mem#:
       <<: *default_task_verify_det
       command: '&LOAD_MODULES_RUN_TASK; "run_vx" "&JOBSdir;/JREGIONAL_RUN_MET_GRIDSTAT_OR_POINTSTAT_VX"'
       envars:
         <<: *default_vars
-        OBS_DIR: '&NDAS_OBS_DIR;'
+        OBS_DIR: '#METAOBS_DIR#'
         FIELD_GROUP: '#FIELD_GROUP#'
         METPLUSTOOLNAME: 'POINTSTAT'
-        OBTYPE: 'NDAS'
+        OBTYPE: '#METAOBTYPE#'
         ENSMEM_INDX: "#mem#"
         SLASH_ENSMEM_SUBDIR_OR_NULL: '{% if global.DO_ENSEMBLE %}{{ "/mem#mem#" }}{% endif %}'
-        OBS_AVAIL_INTVL_HRS: '{{- verification.NDAS_OBS_AVAIL_INTVL_HRS }}'
+        OBS_AVAIL_INTVL_HRS: '#METAOB_AVAIL_HRS#'
         FCST_LEVEL: 'all'
         FCST_THRESH: 'all'
       walltime: 01:00:00
       dependency:
         and:
-          datadep_all_pb2nc_obs_ndas_complete:
+          datadep_all_nc_obs_present:
             attrs:
               age: 00:00:00:30
-            # Check that the flag files that indicate that the Pb2NC tasks are
-            # complete are all present before launching any PointStat task.
+            # Check that the flag files that indicate that the Ascii2NC and/or Pb2NC
+            # output files are all present before launching any PointStat task.
             text: '{%- set num_obs_days = workflow.OBS_DAYS_ALL_CYCLES_INST|length %}
                    {%- set indent = "      " %}
                    {%- set indent_p2 = indent + "  " %}
                    {%- for n in range(0, num_obs_days) %}
                      {%- set yyyymmdd = workflow.OBS_DAYS_ALL_CYCLES_INST[n] %}
                      {%- if n == 0 %}
-                       {{- workflow.WFLOW_FLAG_FILES_DIR ~ "/run_met_pb2nc_obs_ndas_" ~ yyyymmdd ~ "_complete.txt" }}
+                       {{- workflow.WFLOW_FLAG_FILES_DIR ~ "/#METAOBTYPE#_nc_obs_" ~ yyyymmdd ~ "_ready.txt" }}
                      {%- else %}
                        {{- indent ~ "</datadep>\n" }}
                        {{- indent ~ "<datadep age=\"00:00:00:30\">\n" }}
-                       {{- indent_p2 ~ workflow.WFLOW_FLAG_FILES_DIR ~ "/run_met_pb2nc_obs_ndas_" ~ yyyymmdd ~ "_complete.txt" }}
+                       {{- indent_p2 ~ workflow.WFLOW_FLAG_FILES_DIR ~ "/#METAOBTYPE#_nc_obs_" ~ yyyymmdd ~ "_ready.txt" }}
                      {%- endif %}
                      {%- if n != num_obs_days-1 %} {{- "\n" }} {%- endif %}
                    {%- endfor %}'
           datadep_post_files_exist:
             attrs:
               age: 00:00:00:30
             text: !cycstr '{{ workflow.EXPTDIR }}/@Y@m@d@H/post_files_exist_mem#mem#.txt'
+          or:
+            not:
+              taskvalid:
+                attrs:
+                  task: 'run_MET_PcpCombine_fcst_#FIELD_GROUP#_mem#mem#'
+            taskdep:
+              attrs:
+                task: 'run_MET_PcpCombine_fcst_#FIELD_GROUP#_mem#mem#'
```

Comment:
@mkavulich Is this new dependency because one or more of the new obs types (AERONET and AIRNOW) require PcpCombine to be run on the forecast but SFC and UPA do not?

Reply:
This new dependency is needed for PM2.5 coming from the model output; it is split among two different fields, so those must be combined in order to get the correct results from PointStat (see the changes to [PcpCombine.conf](https://github.com/ufs-community/ufs-srweather-app/pull/1174/files#diff-ee331a22c1334355a04e0acf65f14522356435395d2b44efe009989027de8291) and [exregional_run_met_pcpcombine.sh](https://github.com/ufs-community/ufs-srweather-app/pull/1174/files#diff-214ff57ffb0f892cd9b4857bc730e5fa291699aefcb46a0d8e708971e424e9e9))